test/test_utils/__init__.py

```diff
@@ -975,3 +1046,16 @@ def run_cmd_on_container(container_name, context, cmd, executable="bash", warn=F
     return context.run(
         f"docker exec --user root {container_name} {executable} -c '{cmd}'", hide=True, warn=warn, timeout=60
     )
+
+
+def ordered(obj):
+    if isinstance(obj, dict):
+        return sorted((k, ordered(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return sorted(ordered(x) for x in obj)
+    else:
+        return obj
+
+
+def are_json_objects_equivalent(obj1, obj2):
```

Comment:
is this being used anywhere?

Reply:
It's not being used anywhere right now. I added it as a helper function for the `ScanVulnerabilityList` class, but I ended up realizing that I can't do an exact match of the JSON vulnerability lists.

I forgot that I added this, and can remove it now. But this seems like a pretty useful utility function to have.

Reply:
IMO this doesn't have a clear use case at the moment. Let's remove for now so as not to add test_utils that don't have clear usage examples int he codebase